internal/testutil/testutil.go

```diff
@@ -50,7 +49,8 @@ func TestModules(t *testing.T, ctrl *gomock.Controller, id hotstuff.ID, _ hotstu
 
 	synchronizer := mocks.NewMockSynchronizer(ctrl)
 	synchronizer.EXPECT().Start(gomock.Any()).AnyTimes()
-	synchronizer.EXPECT().ViewContext().AnyTimes().Return(context.Background())
+	// TODO: This line was previously used to test ViewContext that was part of Synchronizer in the past. Make sure this removal makes sense.
```

Comment:
From the `synchronizer.ViewContext` doc:
```go
// ViewContext returns a context that is canceled at the end of view.
// If view is nil or less than or equal to the current view, the context will be canceled at the next view change.
```
There are other tests for this function. Its use is now via package style rather than as part of the `Synchronizer` interface, which I think is more idiomatic Go style. This `ViewContext` line herein can be removed.